#!/usr/bin/env python
PACKAGE = "kmm_mapping"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pnt_cnt_req", int_t, 0, "Points required on wall to increment times", 7, 0, 50)
gen.add("times_req", int_t, 0, "Number of times pnt_cnt has to reach requirement for wall to be added", 5, 0, 30)

exit(gen.generate(PACKAGE, "kmm_mapping", "Mapping"))
